subsys/net/ip/net_shell.c

```diff
@@ -3659,11 +3826,60 @@ static int cmd_net_ip_add(const struct shell *shell, size_t argc, char *argv[])
 #else
 		net_if_ipv4_addr_add(iface, &addr, NET_ADDR_MANUAL, 0);
 #endif
+
+		if (argv[3]) {
```

Comment:
Ditto, this belongs to the previous commit. Plus, I believe you should rather check the argument count instead of referencing the argument itself (it may not exist)/

Reply:
Pushed it as part of IPv4 changes. Added arg count checks